tests/integration/test_upload.py

```diff
@@ -26,22 +27,23 @@ def setUp(self):
         self.config = TransferConfig(
             multipart_threshold=self.multipart_threshold)
 
+    def create_file(self, size, name=''):
```

Comment:
Let's call it ``get_input_fileobj()`` or something along those lines. ``create_file`` for me seems like you are intending to create a file each time and in reality for the file-like objects you are just creating a file-like object without writing to disk.